packages/import-maps-generate/src/generate.js

```diff
@@ -0,0 +1,22 @@
+import fs from 'fs';
+import path from 'path';
+import commandLineArgs from 'command-line-args';
+import { generateFromYarnLock } from './generateFromYarnLock';
+import injectToHtmlFile from './injectToHtmlFile';
+
+const optionDefinitions = [{ name: 'inject-to' }];
```

Comment:
do we need to support this? eventually you can just point to the sourcemap file from your `index.html`, temporarily maybe it's easier to handle this in `owc-dev-server` maybe?

Reply:
by the spec it is highly encouraged to inline the map at the top in the html as it is blocking ALL other module requests from even starting - so I would argue such a feature is quite needed

Reply:
wondering if that should not be an optimization in the build? I just like to keep it in a separate file and keep a single source of truth

Reply:
since we recommend running import map generation on postinstall, this creates a lot of git diffs/conflicts. Especially when you clone a new project and don't have everything configured correctly yet. This happens a lot already with yarn/package lockfiles.

keeping it in a separate file helps a lot with that

Reply:
a import map is "locked" to the lock file... so as long as you do not change the lock file the import map will not change... even if you generate it multiple times

so I am not sure what you mean by that?

Reply:
lockfiles are also notorious for changing when not needed, often due to misconfiguration but this happens a lot when random people clone repositories they want to contribute to. since we inject the import maps into the user's index.html I'm worried people end up unintentionally changing the index.html in their merge request, adding a source of confusion.

We can try it out with this though and see how it works out in practice, but a separate file just seems a lot cleaner to me.

Reply:
let's keep it in beta until we know how what is the way forward